article-analyzer/src/main/resources/articles.json

```diff
@@ -1,16 +1,16 @@
 [
   {
-    "id": 1,
+    "id": "123e4567-e89b-12d3-a456-426614174001",
```

Comment:
why did you change it to UUID?

Reply:
Just tought it would be better to use uuid type for id

Reply:
do not modify business data